packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -147,6 +147,13 @@ export interface RotationMultiUserOptions {
    * @default Duration.days(30)
    */
   readonly automaticallyAfter?: Duration;
+
+  /**
+   * Specifies characters to not include in generated passwords.
+   *
+   * @default -
+   */
+  readonly excludedCharacters?: string;
```

Comment:
Is this used anywhere?

Reply:
It is passed with the rest of the options through to the `secretsmanager.SecretRotation()` call at https://github.com/ahammond/aws-cdk/blob/excluded-characters-in-aurora-passwords/packages/%40aws-cdk/aws-rds/lib/cluster.ts#L593